packages/compass-indexes/src/utils/has-columnstore-indexes-support.spec.ts

```diff
@@ -1,24 +0,0 @@
-import { expect } from 'chai';
-
-import { hasColumnstoreIndexesSupport } from './has-columnstore-indexes-support';
-
-describe('hasColumnstoreIndexesSupport', function () {
-  it('returns false for < 7.0.0', function () {
-    expect(hasColumnstoreIndexesSupport('4.2.0')).to.be.false;
```

Comment:
Are we removing this units tests as this is tested in e2e?

Reply:
I was cleaning up and these tests are basically just testing that semver library works, I didn't see much value in testing this. If you have a preference, I can bring them back

Reply:
If that function stops functioning right would it be tested somewhere else?

Reply:
It is still covered by tests through other code:

<img width="852" alt="image" src="https://user-images.githubusercontent.com/5036933/217763893-9d4f499b-b03f-46a9-8c5c-bb138b4a72aa.png">

As I said, if we want to unit test direct dependencies I don't mind that too much even though maintaining these tests is not needed from my perspective. If you have a preference for this, I can bring this file back. Would be nice also then to clarify to what extent do we want to do that: is it every dependency we have, only certain ones, which ones then and how much coverage?

Reply:
Brought back semver tests in 31dc1d54b7a855adcad51545e054fafe7e493272